django_celery_beat/models.py

```diff
@@ -315,11 +315,16 @@ class Meta:
                     'day_of_week', 'hour', 'minute', 'timezone']
 
     def __str__(self):
-        return '{0} {1} {2} {3} {4} {5} (m/h/dM/MY/d) {6}'.format(
-            self.name if self.name else '',
+
+        name = None
+        if self.name:
+            name = '[{}]'.format(self.name)
```

Comment:
Here I wanted to achieve [Name in brackets], but leave an empty '' if there is no name. 
Making if and if - makes it more visible on code, but It could be changed back to 1st version.
Thanks!